src/redux/conference-schedule.js

```diff
@@ -0,0 +1,159 @@
+var keyMirror = require('keymirror');
+var api = require('../mixins/api.jsx').api;
+
+var Types = keyMirror({
+    SET_DAY: null,
+    SET_SCHEDULE: null,
+    SET_SCHEDULE_FETCHING: null,
+    SET_DAY_ERROR: null,
+    SET_SCHEDULE_ERROR: null
+});
+
+module.exports.dayReducer = function (state, action) {
+    if (typeof state === 'undefined') {
+        state = '';
+    }
+    switch (action.type) {
+    case Types.SET_DAY:
+        return action.day;
+    case Types.SET_DAY_ERROR:
+        return state;
+    default:
+        return state;
+    }
+};
+
+module.exports.scheduleReducer = function (state, action) {
+    if (typeof state === 'undefined') {
+        state = [];
+    }
+    switch (action.type) {
+    case Types.SET_SCHEDULE:
+        return action.schedule;
+    case Types.SET_SCHEDULE_FETCHING:
+        return state;
+    case Types.SET_SCHEDULE_ERROR:
+        return state;
+    default:
+        return state;
+    }
+};
+
+module.exports.setDayError = function (error) {
+    return {
+        type: Types.SET_DAY_ERROR,
+        error: error
+    };
+};
+
+module.exports.setDay = function (day) {
+    return {
+        type: Types.SET_DAY,
+        day: day
+    };
+};
+
+module.exports.setSchedule = function (schedule) {
+    return {
+        type: Types.SET_SCHEDULE,
+        schedule: schedule
+    };
+};
+
+module.exports.setScheduleFetching = function () {
+    return {
+        type: Types.SET_SCHEDULE_FETCHING,
+        fetching: true
+    };
+};
+
+module.exports.setScheduleError = function (error) {
+    return {
+        type: Types.SET_SCHEDULE_ERROR,
+        error: error
+    };
+};
+
+module.exports.startGetSchedule = function (day) {
+    return function (dispatch) {
+        dispatch(module.exports.setScheduleFetching());
+        dispatch(module.exports.getDaySchedule(day));
+    };
+};
+
+/**
+ * Gets the schedule for the given day from the api
+ * @param  {String} day  Day of the conference (Thursday, Friday or Satrurday)
+ *
+ *  @return {Object}     Schedule for the day, broken into chunks
+ */
+module.exports.getDaySchedule = function (day) {
+    return function (dispatch) {
+        api({
+            uri: '/conference/schedule/' + day
+        }, function (err, body) {
+            if (err) {
+                dispatch(module.exports.setDayError(err));
+                dispatch(module.exports.setScheduleError(err));
```

Comment:
And if you combine them into one state, then this can just be one error, since they are both always set at the same time.